app/controllers/job_runs_controller.rb

```diff
@@ -1,6 +1,5 @@
 class JobRunsController < ApplicationController
   def index
-    flash[:notice] = 'Success! Your job is queued. A link to your validation report will be emailed to you when it is ready.' if params[:created]
```

Comment:
Instead of passing a param to trigger a flash, we can bank the flash message before the redirect.